examples/step-54/doc/intro.dox

```diff
@@ -53,29 +53,28 @@ U.S. National Bureau of Standards published in 1980 the Initial Graphics
 Exchange Representation (IGES) neutral file format, which is used in this
 example.
 
-<h3> The boundary projector classes </h3>
+<h3> The CAD boundary projector classes </h3>
 
-To import and interrogate CAD models, the deal.II library contains a series of
-wrapper functions for the OpenCASCADE open source library for CAD
-modeling. These functions allow to import IGES files into OpenCASCADE native
-objects, and wrap them inside a series of Manifold classes.
+To import and interrogate CAD models, the deal.II library implements a series of
+wrapper functions for the OpenCASCADE open source library for CAD modeling.
+These functions allow to import IGES files into OpenCASCADE native objects, and
+wrap them inside a series of Manifold classes.
 
 Once imported from an IGES file, the model is stored in a
 <code>TopoDS_Shape</code>, which is the generic topological entity defined in
-the OpenCASCADE framework. From a <code>TopoDS_Shape</code>, it is then
-possible to access all the sub-shapes (such as vertices, edges and faces)
-composing it, along with their geometrical description.  In the deal.II
-framework, the topological entities composing a shape are used to create
-a corresponding Manifold representation. In Step-6 we saw how to build a
-SphericalManifold and assign it to a set of faces (or cells, for co-dimension
-1) of a Triangulation, to have cells and faces refined on a sphere or circle.
-The functions of the CAD modeling interface have been designed to retain the
-same structure, allowing the user to build a projector object using the
-imported CAD shapes, maintaining the very same procedure we use with
-SphericalManifold, i.e., assigning such projector object to cells, faces or
-edges of a coarse mesh. At each refinement cycle, the new mesh nodes will be
-then automatically generated by projecting a midpoint of an existing object
-onto the specified geometry.
+the OpenCASCADE framework. From a <code>TopoDS_Shape</code>, it is then possible
+to access all the sub-shapes (such as vertices, edges and faces) composing it,
+along with their geometrical description. In the deal.II framework, the
+topological entities composing a shape are used to create a corresponding
+Manifold representation. In step-6 we saw how to build a hyper sphere, which is
+automatically assigned a SphericalManifold to its boundary faces, to have them
+refined on a sphere or on a circle. The functions of the CAD modeling interface
```

Comment:
I had a hard time understanding this sentence ("In step-6 we saw how to..."). I think it could be simplified by splitting it into two sentences, perhaps

```
In step-6 we saw how to use GridGenerator::hyper_sphere() to create 
a hyper sphere, which automatically attaches a SphericalManifold
to all boundary faces. This guarantees that boundary faces stay on a
sphere or circle during mesh refinement.
```